scripts/scenarios/cloud6/cloud7-upgrade-non-disruptive-lbaasv2.yml

```diff
@@ -0,0 +1,160 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: libvirt
+      libvirt:
+        hypervisor_ip: ##hypervisor_ip##
+      sbd:
+        nodes:
+          @@controller1@@:
+            devices:
+            - ''
+          @@controller2@@:
+            devices:
+            - ''
+      per_node:
+        nodes:
+          @@controller1@@:
+            params: ''
+          @@controller2@@:
+            params: ''
+    drbd:
+      enabled: true
+  deployment:
+    elements:
+      pacemaker-cluster-member:
+      - @@controller1@@
+      - @@controller2@@
+      hawk-server:
+      - @@controller1@@
+      - @@controller2@@
+- barclamp: nfs_client
+  name: services
+  attributes:
+    exports:
+      glance-images:
+        nfs_server: crowbar
+        export: "/var/lib/glance/images"
+        mount_path: "/var/lib/glance/images"
+        mount_options:
+        - ''
+  deployment:
+    elements:
+      nfs-client:
+      - "@@controller1@@"
+      - "@@controller2@@"
+
+- barclamp: database
+  attributes:
+    ha:
+      storage:
+        mode: drbd
+        drbd:
+          size: 5
+  deployment:
+    elements:
+      database-server:
+      - cluster:services
+- barclamp: rabbitmq
+  attributes:
+    trove:
+      enabled: true
+    ha:
+      storage:
+        mode: drbd
+        drbd:
+          size: 5
+  deployment:
+    elements:
+      rabbitmq-server:
+      - cluster:services
+- barclamp: keystone
+  attributes:
+  deployment:
+    elements:
+      keystone-server:
+      - cluster:services
+- barclamp: glance
+  attributes:
+  deployment:
+    elements:
+      glance-server:
+      - cluster:services
+- barclamp: cinder
+  attributes:
+    volumes:
+    - backend_driver: nfs
+      backend_name: default
+      nfs:
+        nfs_shares: nfsserver:/srv/nfs/cinder
+        nfs_mount_options: ''
+  deployment:
+    elements:
+      cinder-controller:
+      - cluster:services
+      cinder-volume:
```

Comment:
Yeah. Though I didn't spend much thought I why we'd want more here. Would you prefer to have both compute nodes there?